### ISSUE TITLE
Attribute list saving

### ISSUE THREAD

**Issue by meguiraun** [MaxIV-KitsControls/web-maxiv-hdbppviewer#27]
like the old archiver, the user should be able to save the currently displayed attrs and later recover. Everyone sees everyone else's list

**Comment by hardion** [comment]
Just bookmark the Url and it should save the configuration.

**Comment by johanfforsberg** [comment]
Yep, this was the intention from the beginning. All the information to recreate the plot should be in the URL "#..." part, which is read by the client.

I think this is a more useful feature anyway, but of course to work exactly like the old viewer the users also need a way to share bookmarks.
